chapter04/dcox/ex4_3

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+sub total {
+  my $sum = 0;
+
+  $sum += $_ foreach(@_);
+
+  $sum;
+}
+
+sub average {
+  my $avg = 0;
+
+  $avg = &total(@_)/(scalar @_) if(scalar @_);
```

Comment:
I think it will work without `scalar`. In perl, it seems that we can write everything in a single line. lol